### ISSUE TITLE
Feature Request: Namespace Support 

### ISSUE THREAD

**Issue by tiny-dancer** [terraform-providers/terraform-provider-vault#175]
Hi there,

Feature request for namespace support, either via `VAULT_NAMESPACE` or provider configuration hcl block

### Terraform Version
```
Terraform v0.11.7
+ provider.vault v1.1.3
```

### Affected Resource(s)
Provider as a whole

### Terraform Configuration Files
```hcl
provider "vault" {
}

resource "vault_policy" "example" {
  name = "test"

  policy = <<EOT
path "secret/my_app" {
  policy = "write"
}
EOT
}
```

### Expected Behavior
Ability to configure namespace via `VAULT_NAMESPACE` or provider hcl block

### Actual Behavior
Not available 

### References
I couldn't find any references - not sure if this is already tracked?

Thanks!

**Comment by ilapost** [comment]
I've used the Vault provider with namespace support and it seems to work fine. The problem is though that you cannot create namespaces with the provider. Are there any plans to include this functionality?

**Comment by petems** [comment]
Support for namespaces at the provider level were merged in https://github.com/terraform-providers/terraform-provider-vault/pull/262 and https://github.com/terraform-providers/terraform-provider-vault/pull/288

@ilapost Opened a PR for Namespace resource here: https://github.com/terraform-providers/terraform-provider-vault/pull/338

**Comment by ilapost** [comment]
Thank you @petems. I'll keep an eye out for updates.

**Comment by petems** [comment]
Merged and released! 😄

**Comment by thomasbibb** [comment]
@petems Which tagged version?

**Comment by lucymhdavies** [comment]
@thomasbibb  This is included as of v1.6.0 - https://github.com/terraform-providers/terraform-provider-vault/blob/master/CHANGELOG.md#160-march-06-2019 🙂
